{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/adapters.py

```diff
@@ -14,3 +14,13 @@ def is_open_for_signup(self, request: HttpRequest):
 class SocialAccountAdapter(DefaultSocialAccountAdapter):
     def is_open_for_signup(self, request: HttpRequest, sociallogin: Any):
         return getattr(settings, "ACCOUNT_ALLOW_REGISTRATION", True)
+
+    def populate_user(self, request, sociallogin, data):
+        user = sociallogin.user
```

Comment:
Could you please add a docstring to this method explaining what it does? And link to any relevant piece of documentation if such thing exist.

Reply:
```suggestion
        """
        Populates user information from social provider info
        See: https://django-allauth.readthedocs.io/en/latest/advanced.html?highlight=populate_user#creating-and-populating-user-instances
        """
        user = sociallogin.user
```